### PR TITLE
fix: Wrap persistent connection service in try [WPB-11113]

### PR DESCRIPTION
**Pull request by borichellow**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-11113" title="WPB-11113" target="_blank">BugWPB-11113</a>  [Android] crash on starting foreground service
  </td></table>
  <br />
 

# What's new in this PR?

### Issues

crashes in PlayStore https://play.google.com/console/u/0/developers/7098984309886892484/app/4973241010395499500/vitals/crashes/76ed8eaea06144fa56bcc122f39cd8a0/details?days=28&versionCode=100020981

### Causes (Optional)

this happens when PersistentConnection Service is killed by some reason and restarted by OS from background. 

### Solutions

Basically the issue is on Android side, for now google suggests `try-catch` workaround only 
https://issuetracker.google.com/issues/307329994#comment86

Also I've updated `StartPersistentWebsocketIfNecessaryUseCase` to use `ServicesManager` for starting Service instead of doing it directly. So we have 1 place where all the Services started from.

### CODE REVIEW

**Review by damian-kaczmarek — APPROVED** [review]

**Review by MohamadJaara — APPROVED** [review]

**Comment by MohamadJaara** [comment]
the fix version for this one is 4.9 can you please do the cherry pick to RC
